sphinx/util/nodes.py

```diff
@@ -83,7 +85,15 @@ def match(self, node: Node) -> bool:
             return False
 
     def __call__(self, node: Node) -> bool:
-        return self.match(node)
+        return self._match(node)
+
+    def findall_in(self, node: Node) -> Iterator[T]:
```

Comment:
I prefer having something like

```
for node in matcher.findall(...)
```

rather than 

```
for node in matcher.findall_in(...)
```

Main reason is that it's somewhat similar to `re` where you have

```
pattern = re.compile(...)
m = pattern.findall(...)
```

Reply:
done